editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -667,17 +668,22 @@ function useSelectOrLiveModeSelectAndHover(
         (!hadInteractionSessionThatWasCancelled || !draggedOverThreshold.current) &&
         (activeControl == null || activeControl.type === 'BOUNDING_AREA')
 
+      let editorActions: Array<EditorAction> = []
+
       if (event.type === 'mousedown') {
         didWeHandleMouseDown.current = true
       }
       if (event.type === 'mouseup') {
+        const handleMouseUpActions = handleMouseUp(event.nativeEvent)
```

Comment:
to make this code clearer here, I think we should call it `handleGlobalMouseUp` or something similar, so it doesn't look like we are just calling off to a local helper

Reply:
Yeah, that's a better name.

Reply:
Fixed in db2f65791.